openml/flows/flow.py

```diff
@@ -51,10 +52,16 @@ def _generate_flow_xml(self):
             Flow represented as XML string.
         """
         model = self.model
+
+        if config._testmode:
+            flow_name = '%s%s' % (config.testsentinel, self.name)
```

Comment:
I don't like this. There should be no testing code in the main code. Why do you want this?